src/pages/Favorites/Favorites.page.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { useContext } from 'react';
+
+import VideoList from '../../templates/VideoList/VideoList';
+import { ThemeContext } from '../../providers/Theme/Theme.provider';
+import theme from '../../providers/Theme/Theme';
+
+import {
+  ContentMain,
+  TitleDiv,
+  ContDiv,
+  ContDivFst,
+  PrevBtn,
+  NextBtn,
+  NoVideos,
+  NoVideosDiv,
+} from './Favorites.page.styled';
+
+function favStorage() {
+  const values = [];
+  const keys = Object.keys(localStorage);
+
+  for (let i = 0; i < keys.length; i++) {
```

Comment:
This is not the natural way to do the things on React. You should use a `map` here to return the array in the way you want to. Something like:
```
function getFavoritesFromStorage () {
   Object.keys(localStorage).map({key} => {
      return JSON.parse(localStorage.getItem(key))
   }

   )
}
```